File: app/src/main/java/com/paxus/pay/poslinkui/demo/utils/interfacefilter/EntryAction.java
package com.paxus.pay.poslinkui.demo.utils.interfacefilter;

public class EntryAction {
    public String category, action, name, alias;
    boolean enableByDefault;
    public boolean isCurrentlyEnabled = false;

    public EntryAction(String category, String action, String name, String alias, boolean enableByDefault) {
        this.category = category;
        this.action = action;
        this.name = name;
        this.alias = alias;
        this.enableByDefault = enableByDefault;
    }
}

File: app/src/main/java/com/paxus/pay/poslinkui/demo/utils/interfacefilter/EntryAction.kt
package com.paxus.pay.poslinkui.demo.utils.interfacefilter

class EntryAction(
    var category: String?,
    var action: String?,
    var name: String?,
    var alias: String?,
    var enableByDefault: Boolean
) {
    var isCurrentlyEnabled: Boolean = false
}